earthfile2llb/interpreter.go

```diff
@@ -1026,15 +1026,29 @@ func (i *Interpreter) handleEnv(ctx context.Context, cmd spec.Command) error {
 
 var errInvalidSyntax = errors.New("invalid syntax")
 var errRequiredArgHasDefault = errors.New("required ARG cannot have a default value")
+var errGlobalArgNotInBase = errors.New("global ARG can only be set in the base target")
 
 // parseArgArgs parses the ARG command's arguments
 // and returns the argOpts, key, value (or nil if missing), or error
-func parseArgArgs(ctx context.Context, cmd spec.Command) (argOpts, string, *string, error) {
+func parseArgArgs(ctx context.Context, cmd spec.Command, isBaseTarget bool, explicitGlobal bool) (argOpts, string, *string, error) {
```

Comment:
I would name `explicitGlobal` something like `explicitGlobalFeature` - just to make it clear that it represents the feature flag.